packages/@aws-cdk/toolkit-lib/lib/actions/diff/index.ts

```diff
@@ -1,4 +1,4 @@
-import type { StackSelector } from '../../api/cloud-assembly';
+import type { StackSelector } from '@aws-cdk/tmp-toolkit-helpers';
```

Comment:
or whatever the path is
```suggestion
import type { StackSelector } from '../../@aws-cdk/tmp-toolkit-helpers';
```